doc/test_messages_documentation.py

```diff
@@ -53,7 +53,13 @@ def __init__(self, test_file: Tuple[str, Path]) -> None:
         self._linter.config.persistent = 0
         checkers.initialize(self._linter)
 
-        config_file = next(config.find_default_config_files(), None)
+        # Check if this message has a custom configuration file (e.g. for enabling optional checkers).
+        # If not, use the default configuration.
+        config_file: Optional[Path]
```

Comment:
Minor nitpick but I believe this isn't necessary. `mypy` should allow to assign `None` to a previously assigned variable (L60 vs L62).

Reply:
Sorry I merged too fast.

Reply:
I tried it first without this annotation, but ``mypy`` complained.

Reply:
Ah, never mind then!